docs/user_manual.md

```diff
@@ -0,0 +1,76 @@
+# User manual
+
+## Contents
+
+- [Installation](#installation)
+  - [Dependencies](#dependencies)
+  - [Windows](#windows)
+
+## Installation
+
+### Dependencies
+
+- Python (3.8, 3.9)
+- [`virtualenv`](https://virtualenv.pypa.io/en/latest/) (optional, but recommended)
+- [Git](https://git-scm.com/downloads)
+
+### Windows
+
+**Extra Dependencies:**
+
+- [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
```

Comment:
For what?

Reply:
`lap` it seems.  `winget install Microsoft.VisualStudio.2019.BuildTools` did not do the trick.

Reply:
winget isn't built into default windows quite yet either, so we shouldn't use it in our docs quite yet.

Reply:
Looks to be a option to remove `lap` from sort, but unsure what the performance consequence would be.
https://github.com/R0flcopt3r/sort/blob/master/sort/sort.py#L28

Reply:
I will do some benchmarks on that tomorrow.

Reply:
LAP and sicpy uses roughly the same amount of time, see fabc031920db883f1b9ae15b842c11e40b42fd2a

Reply:
With the recent change to `master`, is this dep still necessary?